core/src/main/java/com/graphhopper/routing/util/AbstractFlagEncoder.java

```diff
@@ -181,6 +178,8 @@ public int defineRelationBits(int index, int shift) {
         return shift;
     }
 
+    public abstract boolean acceptsRelation(OSMTurnRelation relation);
```

Comment:
Maybe we should provide a default implementation (returning true) here, such that already existing flag encoders (in other projects) do not need to be updated to keep the same behavior.

Reply:
ok